components/eam/src/physics/rrtmg/cloud_rad_props.F90

```diff
@@ -104,8 +107,10 @@ subroutine cloud_rad_props_init()
    i_lambda = pbuf_get_index('LAMBDAC',errcode=err)
    i_iciwp  = pbuf_get_index('ICIWP',errcode=err)
    i_iclwp  = pbuf_get_index('ICLWP',errcode=err)
-   i_des    = pbuf_get_index('DES',errcode=err)
-   i_icswp  = pbuf_get_index('ICSWP',errcode=err)
+!   if (microp_scheme.ne.'P3') then !AaronDonahue TODO: P3 doesn't have snow
```

Comment:
This isn't used in SCREAM, we should take the upstream version of this file in the merge.